### ISSUE TITLE
[Feature request] - Add arguments to bbc_style() for easy customization

### ISSUE THREAD

**Issue by edgararuiz-zz** [bbc/bbplot#8]
Windows machines do not support Helvetica font type, so I had to make modifications to the `bbc_style()` in a separate `ggplot2()` function.  It would be cool to have maybe a `font` argument that defaults to Helvetica, but would allow others to change the font name.  Once the font changes, there may be some sizing differences, so it would also be cool to create an `offset` size, to do a plus or minus the current font sizes used in your function:  `bbc_style <- function(font = "Helvetica", offset = 0)`, which would allow me to use it like this: `bbc_style("Arial", -2)`.

**Comment by jlopezper** [comment]
@edgararuiz could you please check whether my PR fits your request?

**Comment by edgararuiz-zz** [comment]
Yes! I just re ran my script using your fork and it worked like a charm. Thanks!

**Comment by atusy** [comment]
I think `bbc_style` better have arguments consistent to ggplot2 `theme_*` functions like below, and sizes being controled by `ggplot2::rel()`.

```
bbc_style <- function(
  base_size, 
  base_family = "Helvetica", 
  base_line_size, 
  base_rect_size, 
  offset = 0
)
```
